[docs] Document CSV steps (#4810)
vividus-framework/vividus: docs/modules/plugins/pages/plugin-csv.adoc

----
{transformer=FROM_CSV, csvPath=/data/another-example.csv, delimiterChar=;}
----

== Steps

=== Save CSV into variable

> Save CSV string to indexed zero-based variable with map structure, e.g. var[0].key2, var[1].key1 etc. Please see xref:commons:variables.adoc#_complex_types_variables[documentation] for more information about complex variables.

Reviewer: ```suggestion
Saves CSV string to indexed zero-based variable with column name mappings, e.g. var[0].key2, var[1].key1 etc. Please see thexref:commons:variables.adoc#_complex_types_variables[documentation] for more information about complex variables.
```

[source,gherkin]
----
When I save CSV `$csv` to $scopes variable `$variableName`
----

. `$csv` - CSV string.

> . `$scopes` - xref:commons:variables.adoc#_scopes[The comma-separated set of the variables scopes].

Reviewer: ```suggestion
. `$scopes` - The comma-separated set of the xref:commons:variables.adoc#_scopes[variables scopes].
```

> . `$variableName` - A name of variable to save CSV string.

Reviewer: ```suggestion
. `$variableName` - The name of the variable to save CSV string.
```

> .Validate CSV

Reviewer: ```suggestion
.Validate CSV data
```
